NOTICE.txt

```diff
@@ -82,3 +82,8 @@ its NOTICE file:
 
   This product includes software developed by Hewlett-Packard:
   (c) Copyright [2014-2015] Hewlett-Packard Development Company, L.P
+
+---------------------------------------------------------------------------------
+
+This product includes software from The MathWorks, Inc. (Apache 2.0)
+  * Copyright (C) 2024 The MathWorks, Inc.
```

Comment:
We need this only for `NOTICE.txt` in `.mltbx` because our source archive doesn't include software from The MathWorks, Inc. See also: https://infra.apache.org/licensing-howto.html#binary

Can we add this in `/dev/tasks/matlab/github.yml` dynamically instead of adding this to `/NOTICE.txt` statially? We use `/NOTICE.txt` for source archive too.

Reply:
Updated the `packageMATLABInterface.m` to dynamically append the required text to packaged copies of `NOTICE.txt` and `LICENSE.txt`